migrations/2_alter_project_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex, Promise) {
+  return Promise.join(
+    knex.schema.table('projects', function (t) {
+      t.boolean('readonly');
```

Comment:
Clearly I missed something in this last week or so. It looks like we're trying to separate management of the database we're using for publish from the database we use to test publish.

If so, I'd prefer to see that land before this. Is there an issue on this (rather large) change to the workflow?